packages/widget-github/src/Widget.tsx

```diff
@@ -60,201 +60,195 @@ let Github = () => {
     return filteredTrends;
   }, [trends, trendFilter]);
 
-  return (
-    <>
-      <Grid item xs={1} style={{ maxWidth: "100%" }}>
-        <div className={classes.widgetTitle}>
-          <Grid
-            container
-            direction="row"
-            wrap="nowrap"
-            alignItems="stretch"
-            alignContent="stretch"
-            justifyContent="space-between"
-          >
-            <Grid item>
-              <Typography variant="subtitle1">Trending on Github</Typography>
-            </Grid>
-            <Grid item>
-              <Grid container direction="row" spacing={1}>
-                <Grid item>
-                  <Filter />
-                </Grid>
-                <Grid item>
-                  <TrendingDialogLauncher />
-                </Grid>
-                <Grid item>
-                  <HidePop name="github" />
-                </Grid>
-                <Grid item>
-                  <Dragger />
-                </Grid>
-              </Grid>
-            </Grid>
-          </Grid>
-        </div>
-      </Grid>
-      <Grid item xs>
+  return <>
+    <Grid item xs={1} style={{ maxWidth: "100%" }}>
```

Comment:
Can we revert this indentation change? I would make it easier to review the file.